src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/__tests__/HealthCertificateValidationServiceValidationTests.swift

```diff
@@ -299,775 +169,50 @@ class HealthCertificateValidationServiceValidationTests: XCTestCase {
-		
-		let vaccinationValueSetsProvider = VaccinationValueSetsProvider(
-			client: CachingHTTPClientMock(),
-			store: store
-		)
-		
-		let validationResults = [
-			ValidationResult(rule: Rule.fake(identifier: "Rule A"), result: .passed),
-			ValidationResult(rule: Rule.fake(identifier: "Rule B"), result: .open),
-			ValidationResult(rule: Rule.fake(identifier: "Rule C"), result: .fail)
-		]
-		
-		var validationRulesAccess = MockValidationRulesAccess()
-		validationRulesAccess.expectedAcceptanceExtractionResult = .success([])
-		validationRulesAccess.expectedInvalidationExtractionResult = .success([])
-		validationRulesAccess.expectedValidationResult = .success(validationResults)
-		
-		let rulesDownloadService = RulesDownloadService(validationRulesAccess: validationRulesAccess, signatureVerifier: MockVerifier(), store: store, client: client)
-
-		let validationService = HealthCertificateValidationService(
-			store: store,
-			client: client,
-			vaccinationValueSetsProvider: vaccinationValueSetsProvider,
-			validationRulesAccess: validationRulesAccess,
-			dccSignatureVerifier: DCCSignatureVerifyingStub(),
-			dscListProvider: dscListProvider,
-			rulesDownloadService: rulesDownloadService
-		)
-		
-		// expirationTime must be >= validation clock to succeed.
-		let expirationTime = Date(timeIntervalSince1970: 1625655530)
-		let validationClock = Date(timeIntervalSince1970: TimeInterval(0))
-		
-		let healthCertificateBase45 = DigitalCovidCertificateFake.makeBase45Fake(
-			from: DigitalCovidCertificate.fake(
-				name: .fake(familyName: "Brause", givenName: "Pascal", standardizedFamilyName: "BRAUSE", standardizedGivenName: "PASCAL"),
-				testEntries: [TestEntry.fake(dateTimeOfSampleCollection: "2021-06-06T06:06:06Z")]
-			),
-			and: CBORWebTokenHeader.fake(expirationTime: expirationTime)
-		)
-		
-		guard case let .success(base45) = healthCertificateBase45 else {
-			XCTFail("Could not create fake health certificate. Abort test.")
-			return
-		}
-		let healthCertificate = try HealthCertificate(base45: base45)
-		
-		let expectation = self.expectation(description: "Test should success with .validationFailed")
-		var responseReport: HealthCertificateValidationReport?
-		
-		// WHEN
-		validationService.validate(
-			healthCertificate: healthCertificate,
-			arrivalCountry: try country(),
-			validationClock: validationClock,
-			completion: { result in
-				switch result {
-				case let .success(report):
-					responseReport = report
-					expectation.fulfill()
-				case let .failure(error):
-					XCTFail("Test should not fail with error: \(error)")
-				}
-			}
-		)
-		
-		// THEN
-		waitForExpectations(timeout: .short)
-		guard let report = responseReport else {
-			XCTFail("report must not be nil")
-			return
-		}
-		XCTAssertEqual(report, .validationFailed(validationResults))
-	}
-	
-	// MARK: - Errors (ValueSets)
-	
-	func testGIVEN_ValidationService_WHEN_expirationDateHasReached_THEN_TECHNICAL_VALIDATION_FAILED_IsReturned() throws {
```

Comment:
Same question here, maybe also for the following tests. The tests look like they are testing `HealthCertificateValidationService` and not a RestService behavior. Are these test cases now somewhere else?

Reply:
Same here again: The test is still there
https://github.com/corona-warn-app/cwa-app-ios/blob/d3feafff70f31847196b61763b3fa8a22ea0d40e/src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/__tests__/HealthCertificateValidationServiceValidationTests.swift#L274